server/src/main/java/org/elasticsearch/gateway/PersistedClusterStateService.java

```diff
@@ -219,6 +222,9 @@ public static void deleteAll(Path[] dataPaths) throws IOException {
 
     // exposed for tests
     Directory createDirectory(Path path) throws IOException {
+        if (USE_RAM_DIR) {
+            return new RAMDirectory();
```

Comment:
Can you use the new `ByteBuffersDirectory` which is the replacement for the deprecated `RAMDirectory` ?